monash-human-power/MHP-website
# Transition from NPM to Yarn

## Description
NPM is really slow to update packages and install the website. Yarn is much quicker and easier to use.

## Steps to Test
1. Install yarn
2. Run `yarn install`
3. Run `yarn run develop`
4. check that the development server runs correctly
